config/fcos/v1_6_exp/validate.go

```diff
@@ -22,6 +22,21 @@ import (
 	"github.com/coreos/vcontext/report"
 )
 
+const rootDevice = "/dev/disk/by-id/coreos-boot-disk"
+
+func (conf Config) Validate(c path.ContextPath) (r report.Report) {
+	for i, disk := range conf.Storage.Disks {
+		if disk.Device != rootDevice && (util.IsFalse(disk.WipeTable) || disk.WipeTable == nil) {
+			for p, partition := range disk.Partitions {
+				if partition.Number == 0 {
+					r.AddOnWarn(c.Append("storage", "disks", i, "partitions", p), common.ErrInvalidPartitionConfig)
```

Comment:
Since we're warning specifically about the `number` field, let's add it to the path rather than pointing generically to the entire partition section.

Reply:
Good point have that pushed now!